recipe/emailverification/utils.go

```diff
@@ -34,6 +34,7 @@ func validateAndNormaliseUserInput(appInfo supertokens.NormalisedAppinfo, config
 	typeNormalisedInput.GetEmailDeliveryConfig = func() emaildelivery.TypeInputWithService {
 		createAndSendCustomEmail := DefaultCreateAndSendCustomEmail(appInfo)
 		if config.CreateAndSendCustomEmail != nil {
+			supertokens.LogWarningMessage("CreateAndSendCustomEmail is deprecated. Use EmailDelivery instead.")
```

Comment:
This is not the right way to mark as deprectaed

You need to do how it's done here: https://github.com/supertokens/supertokens-golang/blob/emailpassword-email-service/recipe/emailverification/evmodels/models.go#L26